src/components/NcContent/NcContent.vue

```diff
@@ -222,6 +222,8 @@ export default {
 
 .content {
 	box-sizing: border-box;
+	margin: var(--body-container-margin);
+	margin-top: var(--header-height);
```

Comment:
It breaks nccontent for apps :pensive: 
Tried it with my vue3 version of forms.

And included here as it otherwise break visual tests :sweat_smile:

Reply:
> It breaks nccontent for apps

Having it or not having breaks apps?

Reply:
> Having it or not having breaks apps?

Not having it. Have you tried running a Vue3 app? Without this the app content is shown underneath the header menu as the margin is missing.

Reply:
> Have you tried running a Vue3 app?

Actually, we tried. Talk worked on Vue 3 and I don't remember issues with NcContent margins…

Reply:
Adding these two lines will break the layout. As I tried to explain in #4831, in vue3 `<main id="content">` does not get replace by `<div id="content-vue">`, but `<div id="content-vue"> gets mounted within. If both have a margin, we will end up with a gap. See the screenshot below for how this would look like:

![grafik](https://github.com/user-attachments/assets/4a08a6a7-0e8d-4c57-adbc-68599a239eab)

Reply:
Oh good point I tested it with some kind of mix of nextcloud-vue and legacy HTML only app.
But with only vue you are right.

Reverted this as I think everyone will use:
```
const app = createApp(YourVueAppView)
app.mount('#content')
```